src/Entities/PaymentInstrumentValidation.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * This source file is proprietary and part of Rebilly.
+ *
+ * (c) Rebilly SRL
+ *     Rebilly Ltd.
+ *     Rebilly Inc.
+ *
+ * @see https://www.rebilly.com
+ */
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Resource;
+
+class PaymentInstrumentValidation extends Resource
```

Comment:
Maybe it makes sense to rename to `PaymentCardValidation`. (maybe extends abstract `PaymentInstrumentValidation`). Because almost all getters in this class will be confused if we will add other type of validation.